plugins/poll/assets/javascripts/widgets/discourse-poll.js.es6

```diff
@@ -415,6 +422,211 @@ createWidget("discourse-poll-info", {
   }
 });
 
+function fetchGroupedResults(data) {
+  return ajax("/polls/grouped_poll_results.json", { data }).catch(error => {
+    if (error) {
+      popupAjaxError(error);
+    } else {
+      bootbox.alert(I18n.t("poll.error_while_fetching_voters"));
+    }
+  });
+}
+
+function transformUserFieldToLabel(fieldName) {
+  let transformed = fieldName.split("_");
+  transformed[0] =
```

Comment:
This seems to be doing a `classify` operation on the string? Might as well use Ember's version!

Reply:
You can still use the `split` and `join`, but with `classify` in between.